html/semantics/forms/form-submission-0/constructing-form-data-set.html

```diff
@@ -46,4 +47,64 @@
   form2.submit.click();
 }, 'The button cannot be setted if it is not a submitter.');
 
+test(() => {
+  let didCallHandler = false;
+  let wasBubbles = false;
+  let wasCancelable = true;
+  let form = populateForm();
+  document.addEventListener('formdata', e => {
+    didCallHandler = true;
+    wasBubbles = e.bubbles;
+    wasCancelable = e.cancelable;
+  });
+  new FormData(form);
+  assert_true(didCallHandler);
+  assert_true(wasBubbles);
+  assert_false(wasCancelable);
+}, '"formdata" event bubbles, and is not cancelable.');
+
+test(() => {
+  let didCallHandler = false;
+  let form = populateForm();
+  let orphanRoot = document.createElement('div');
+  orphanRoot.appendChild(form);
+  orphanRoot.addEventListener('formdata', e => {
+    didCallHandler = true;
+  });
+  new FormData(form);
+  assert_true(didCallHandler);
```

Comment:
Same as above, I guess... I'm reading this section:
https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#constructing-form-data-set:formdataevent

And the event fires as normal, and `bubbles = false` per the `EventInit` dict.